README.md

```diff
@@ -72,21 +72,22 @@ Create a `.env` file based on `.env.schema` located on the root folder and set t
+| `DB_PORT`                   | Database Port                                                                                                                                                                                                                                                                                        |                                        |
+| `DB_USER`                   | Database User                                                                                                                                                                                                                                                                                        |                                        |
+| `ID_CUSTOM_ALPHABET`        | Custom Alphabet for local ID generation                                                                                                                                                                                                                                                              | '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ' |
+| `ID_CUSTOM_SIZE`            | Custom size of ID for local ID generation                                                                                                                                                                                                                                                            | 21                                     |
+| `ID_USELOCAL`               | Generate ID locally                                                                                                                                                                                                                                                                                  | true                                   |
+| `LECTERN_URL`               | Schema Service (Lectern) URL                                                                                                                                                                                                                                                                         |                                        |
+| `LOG_LEVEL`                 | Log Level                                                                                                                                                                                                                                                                                            | 'info'                                 |
+| `PLURALIZE_SCHEMAS_ENABLED` | This feature automatically convert schema names to their plural forms when handling compound documents                                                                                                                                                                                               | true                                   |
```

Comment:
New environment variable. When enabled it automatically pluralize schema names on compound documents